mirgecom/simutil.py

```diff
@@ -108,23 +103,25 @@ def get_sim_timestep(discr, state, t, dt, cfl, eos,
     cfl: float
         The current CFL number
     eos: :class:`~mirgecom.eos.GasEOS`
-        Gas equation-of-state supporting speed_of_sound
+        Gas equation-of-state supporting speed_of_sound, and, optionally for viscous
+        fluids, a non-empty :class:`~mirgecom.transport.TransportModel` for viscous
+        transport properties.
```

Comment:
Deriving from `GasEOS` implies that it supports `speed_of_sound` (or `sound_speed`, whatever), so it shouldn't be necessary to say this explicitly.

Reply:
(This one is still here BTW.)

Reply:
Yikes!